node/packages/parser/src/ethereum/yearn.ts

```diff
@@ -28,7 +45,9 @@ export class Parser implements GenericParser {
     const receiveAddress = tx.vout?.[0].addresses?.[0]
 
     if (!this.yearnTokenVaultAddresses) {
-      this.yearnTokenVaultAddresses = await getYearnTokenVaultAddresses()
+      this.yearnTokenVaultAddresses = await this.yearnSdk?.vaults
+        .get()
+        .then((result) => result.map((vault) => vault.address))
```

Comment:
await/then feels odd to me a bit
```suggestion
      const vaults = await this.yearnSdk?.vaults.get()
      this.yearnTokenVaultAddresses = vaults?.map((vault) => vault.address)
```